src/Storages/System/StorageSystemParts.cpp

```diff
@@ -45,54 +45,54 @@ namespace DB
-        {"partition_id",                                std::make_shared<DataTypeString>()},
-        {"min_block_number",                            std::make_shared<DataTypeInt64>()},
-        {"max_block_number",                            std::make_shared<DataTypeInt64>()},
-        {"level",                                       std::make_shared<DataTypeUInt32>()},
-        {"data_version",                                std::make_shared<DataTypeUInt64>()},
-        {"primary_key_bytes_in_memory",                 std::make_shared<DataTypeUInt64>()},
-        {"primary_key_bytes_in_memory_allocated",       std::make_shared<DataTypeUInt64>()},
-        {"is_frozen",                                   std::make_shared<DataTypeUInt8>()},
-
-        {"database",                                    std::make_shared<DataTypeString>()},
-        {"table",                                       std::make_shared<DataTypeString>()},
-        {"engine",                                      std::make_shared<DataTypeString>()},
-        {"disk_name",                                   std::make_shared<DataTypeString>()},
-        {"path",                                        std::make_shared<DataTypeString>()},
-
-        {"hash_of_all_files",                           std::make_shared<DataTypeString>()},
-        {"hash_of_uncompressed_files",                  std::make_shared<DataTypeString>()},
-        {"uncompressed_hash_of_compressed_files",       std::make_shared<DataTypeString>()},
-
-        {"delete_ttl_info_min",                         std::make_shared<DataTypeDateTime>()},
-        {"delete_ttl_info_max",                         std::make_shared<DataTypeDateTime>()},
-
-        {"move_ttl_info.expression",                    std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
-        {"move_ttl_info.min",                           std::make_shared<DataTypeArray>(std::make_shared<DataTypeDateTime>())},
-        {"move_ttl_info.max",                           std::make_shared<DataTypeArray>(std::make_shared<DataTypeDateTime>())},
+    ColumnsDescription{
+        {"partition",                                   std::make_shared<DataTypeString>(),    "The partition name."},
+        {"name",                                        std::make_shared<DataTypeString>(),    "Name of the data part."},
+        {"uuid",                                        std::make_shared<DataTypeUUID>(),      "The UUID of data part."},
+        {"part_type",                                   std::make_shared<DataTypeString>(),    "The data part storing format. Possible Values: Wide (a file per column) and Compact (a single file for all columns)."},
+        {"active",                                      std::make_shared<DataTypeUInt8>(),     "Flag that indicates whether the data part is active. If a data part is active, it's used in a table. Otherwise, it's deleted. Inactive data parts remain after merging."},
```

Comment:
```suggestion
        {"active",                                      std::make_shared<DataTypeUInt8>(),     "Flag that indicates whether the data part is active. If a data part is active, it's used in a table. Otherwise, it's about to be deleted. Inactive data parts appear after merging and mutating operations."},
```